### ISSUE TITLE
Accessibility: Improve contrast

### ISSUE THREAD

**Issue by aparkening** [alistapart/AListApart#218]
From Andrew's accessibility audit:

There are several items with too-low contrast.

- Dateline text at the bottom of each item under “more from A List Apart” is too low contrast.  3.54:1 and needs to be 4.5:1 to meet WCAG 2.0 criteria. (“Item-summary-footer” and “item-summary-footer a” classes should be adjusted.)
![low-contrast-date](https://cloud.githubusercontent.com/assets/7241111/7903626/cc36898e-07b0-11e5-87a9-0568b60b664b.png)

- Figure indicator that appears when an image in an article is hovered (why do we do this?) is very light.
![low-contrast-figure](https://cloud.githubusercontent.com/assets/7241111/7903649/02755458-07b1-11e5-87a4-4c92343f7f38.png)

- Buttons: previous button, embed button, close button in embed feature

- Text of comment when embed feature is activated <div style="opacity: 0.1;" class="comment-body">
![low-contrast-embed](https://cloud.githubusercontent.com/assets/7241111/7903653/1d5e595e-07b1-11e5-8b67-c86de3703c25.png)

This is a WCAG 2.0 1.4.3 failure

**Comment by aparkening** [comment]
As discussed, we won't change the last bullet point.  We'll leave the comment faded when activating the embed code, as readers aren't *reading* the comment at this point.

**Comment by mikepick** [comment]
Let's make the dateline #666666.

The figure text could be #666666 as well, although maybe it bears discussion as to how we could do this feature better.

previous, embed, close buttons let's make #666666 background and boost to #444444 on hover.

#666666 will give us 5.74:1 contrast on white according to what I can find.
